scripts/create_new_jsons.py

```diff
@@ -0,0 +1,271 @@
+"""create_new_jsons
+
+This script is used to create new json entries for new psu versions.
+This script will also update json indexes for the new psu versions.
+
+Note: YOU WILL STILL NEED TO:
+    1. Run sign.sh to sign the new psu version jsons
+    2. Ensure that the filenames for the `aqavit_results_link` enties are correct
+
+Requirements:
+    1. python 3.8 and higher
+
+Usage:
+  create_new_jsons --folder=<str> --new_version=<str> --psu_tag=<str>
+
+Options:
+    --folder=<str>          Folder containing the json files
+    --new_version=<str>     New version to be added
+    --psu_tag=<str>         Tag for the new version
+    --help                  Show this help message
+Example:
+  create_new_jsons --folder=../11 --new_version=11.0.26+4 --psu_tag=jan-2025-psu
+"""
+
+import json
+import logging
+import sys
+from datetime import datetime
+from pathlib import Path
+from typing import Tuple
+import requests
+
+from docopt import docopt  # type: ignore
+
+logging.basicConfig(format="%(message)s", level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+DATE_FORMAT = "%Y-%m-%dT%H:00:00Z"
+
+
+def create_new_json_file(
+    folder: str,
+    new_version: str,
+) -> str:
+    """Create a new JSON file based on an existing one.
+    Args:
+        folder (str): The folder containing the JSON files.
+        new_version (str): The new version to be added.
+    Returns:
+        new_file_path (str): The path to the new JSON file.
+    """
+    # Read the existing JSON file
+    index_file_path = Path(folder).joinpath("index.json")
+    with index_file_path.open("r") as index_file:
+        index_data = json.load(index_file)
+
+    # Get the filename from the "releases" key
+    old_filename = index_data["releases"][0]
```

Comment:
NIT. We are assuming that the first entry is going to be the latest one. Is there a way to be sure we are grabbing the latest entry? (In case someone adds a version manually)

Reply:
The first entry will always be the latest one. This is enforced in the repo

Reply:
Note: I am no longer reading the index file for the most recent json file name, so we no longer need to discuss this method